.github/ISSUE_TEMPLATE/component-or-pattern.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Component or pattern 💡
+about: Create a component or pattern for this project
+labels: 'status: needs triage'
+---
+
+### Summary
+
+> Summarise the component or pattern in one or two sentences.
+
+### Details
+
+> Link to design guidance.
+
+### Additional assets
+
+> Link to additional assets, for example, existing code, images, and
+> prototypes - provide as much useful information as you can.
+
+### Collaborators
+
+<!--
+  Add a name in each of the options below, for example:
+  - Design owner - John Smith
+-->
+
+- Design owner(s) -
+- Development peer reviewer(s) -
+- Sponsor user(s) -
+
+### Definition of done
+
+- [ ] Delivery follows the guidance in
+      [Contributing Guide](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/CONTRIBUTING.md#5-make-a-pull-request)
+- [ ] Feedback from collaborators is addressed, and any follow-up issues are
+      created
+
+#### Testing
+
+- [ ] Tests are behavior-driven and based on interactions defined in the design
+- [ ] Inputs and outputs are tested
+- [ ] Accessibility tests for each use case
```

Comment:
Might be helpful if we could provide an example of how to run this accessibility test. Or if this test is to be written for each component, an example of one in use.